SocialFilter/sections/specific.txt

```diff
@@ -48,6 +48,7 @@ sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 ! ##.content__post--meta-social
 otomobilgunluklerim.com,tilki.net,ekosayfa.com,bilisimogretmeni.com,azbuz.org,siberkalem.com,teknobird.com,buneymis.net,teknoxhaber.com,fosforludusunceler.com,kirtkirtla.com,oyuncukuru.org,denizcilikbilgileri.com,hknkr.com,kadioglublog.com##.content__post--meta-social
 !
+honto.jp##.stSns01
```

Comment:
Is this typo? On my end it's `##.stSnsList01`.

Reply:
Sorry.
I had only added the rules for PC.
I forgot the rules for mobile.
`##.stSnsList01` is for mobile rule.

Reply:
<details><summary>mobile</summary>

![](https://i.ibb.co/VJYvSvS/iphone.png)
</details><br/>


<details><summary>PC</summary>

![](https://i.ibb.co/VxXkq0F/pc.png)
</details><br/>

Reply:
You're right.